docs/events.md

```diff
@@ -0,0 +1,88 @@
+# Player Events
+A string representing an event within the player.
+
+## Overview
+
+The player events are consisted of two event types:
+
+ 1. HTML5 events - various events sent by the browser when handling media embedded using the `<video>` element.
+ 2. Kaltura player custom events - special events indicating changes in the state of the player that does not exist in the html5 video event list and are related to the integral behavior of the player. Such as ads, fullscreen and subtitles events.
+
+Only DOM elements may be true EventTargets. Inside the player, we use fake events targets (fake-event-target.js), and fake events (fake-event) in order to dispatch events not from a DOM element.
+
+## Importing the event list
+
+ ```javascript
+ import {HTML5_EVENTS as Html5Events, CUSTOM_EVENTS as CustomEvents} from 'path/to/../event/events'
+```
+
+
+
+## HTML5 events list
+| Event Name | Event String | Description |
+|--|--|--|
+| ABORT | abort | Fires when the loading of an audio/video is aborted
+| CAN_PLAY | canplay | Fires when the browser can start playing the audio/video
+|CAN_PLAY_THROUGH | canplaythrough | Fires when the browser can play through the audio/video without stopping for buffering|
+|DURATION_CHANGE | durationchange| Fires when the duration of the audio/video is changed|
+| EMPTIED|emptied | Fires when the current playlist is empty|
+|ENDED | ended| Fires when the current playlist is ended|
+| ERROR| error| Fires when an error occurred during the loading of an audio/video|
+| LOADED_DATA| loadeddata | Fires when the browser has loaded the current frame of the audio/video
+|  LOADED_METADATA| loadedmetadata |Fires when the browser has loaded meta data for the audio/video
+|  LOAD_START| loadstart|Fires when the browser starts looking for the audio/video
+|  PAUSE| pause |Fires when the audio/video has been paused
+|PLAY  | play |Fires when the audio/video has been started or is no longer paused
+|  PLAYING| playing |Fires when the audio/video is playing after having been paused or stopped for buffering
+| PROGRESS | progress|Fires when the browser is downloading the audio/video
+| RATE_CHANGE | ratechange |Fires when the playing speed of the audio/video is changed
+| SEEKED | seeked |Fires when the user is finished moving/skipping to a new position in the audio/video
```

Comment:
Fires when the user has finished moving/skipping to a new position in the audio/video